### PR TITLE
Add skeleton file for Chicago

### PR DESCRIPTION
**Pull request by githubteacher**
Headers of where to eat and what to do @crichID what do you think?

### CODE REVIEW

File: githubteacher-chicago.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Chicago, IL
+
+### Places to Eat
```
Comment (crichID) [inline githubteacher-chicago.md:3]: @githubteacher can you recommend the best place to get Chicago Deep Dish Pizza?

File: githubteacher-chicago.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+### Things to Do
```
Comment (crichID) [inline githubteacher-chicago.md:7]: Where is the best place to go train-spotting?

**Review by crichID — CHANGES_REQUESTED** [review]
@githubteacher I added a few special requests!

File: githubteacher-chicago.md (hunk 1 above)
Comment (githubteacher) [reply githubteacher-chicago.md:3]: Oooh yes! 🍕

**Review by D1441 — DISMISSED** [review]
Request changes.

File: githubteacher-chicago.md (hunk 1 above)
Comment (sssrose) [reply githubteacher-chicago.md:3]: Pizza Pizza 👍

**Comment by githubteacher** [comment]
@crichID @D1441 Look good to go now? 🚢 :shipit:

**Review by crichID — APPROVED** [review]

**Pull request merged**